apps/reference/nav/_referenceSidebars.js

```diff
@@ -4,7 +4,7 @@ const sidebars = {
       type: 'category',
       label: 'Overview',
       collapsed: true,
-      items: ['about', 'guides/features', 'architecture', 'guides/examples'],
```

Comment:
is this correct? We don't want the examples page anymore?

Reply:
I think in its current state it's more overwhelming rather than useful. Wdyt?

Reply:
I remember feeling the same when I first came across it. Do we need to add a redirect for it or just taking it out of the sidebar temporarily?